### PR TITLE
#139 support DPUB

### PR DESCRIPTION
**Pull request by iameugenejo**

### CODE REVIEW

File: nsq/writer.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def get(self):
 
                 self.nsq.pub(topic, msg) # pub
                 self.nsq.mpub(topic, [msg, msg_cn]) # mpub
+                self.nq.dpub(topic, 60, msg) # dpub
```
Comment (mreiferson) [inline nsq/writer.py]: `nsq` typo

File: nsq/writer.py [hunk 2]
```diff
@@ -131,7 +135,7 @@ def _pub(self, command, topic, msg, callback):
         conn.callback_queue.append(callback)
         cmd = getattr(protocol, command)
         try:
-            conn.send(cmd(topic, msg))
+            conn.send(cmd(topic, msg) if delay_ms is None else cmd(topic, delay_ms, msg))
```
Comment (mreiferson) [inline nsq/writer.py]: this is unfortunate, we could probably clean it up, but it doesn't matter that much I guess

**Comment by mreiferson** [comment]
@iameugenejo thanks for putting this together!

File: nsq/writer.py (hunk 2 above)
Comment (iameugenejo) [inline nsq/writer.py]: I tried my best not to modify much from original code. I can try cleaning it up.

**Comment by mreiferson** [comment]
LGTM, would you mind squashing down to one commit?

Thanks!

**Comment by iameugenejo** [comment]
The commit has been squashed.

Thank you!

**Pull request merged**